weblayer/renderer/content_renderer_client_impl.cc

```diff
@@ -96,9 +96,14 @@ void ContentRendererClientImpl::RenderThreadStarted() {
   weblayer_observer_ = std::make_unique<WebLayerRenderThreadObserver>();
   thread->AddObserver(weblayer_observer_.get());
 
+  // Broker is important for IPC (because render processes are heavily sandboxed.)
```

Comment:
I would delete this comment. Since this is in Chromium code, you are potentially creating more merge headaches down the road.